ocs_ci/ocs/resources/storage_cluster.py

```diff
@@ -624,6 +626,23 @@ def osd_encryption_verification():
             raise ValueError("OSD is not encrypted")
 
 
+def taint_node_verification():
+    """
+    Verify worker nodes are tainted
+
+    """
+    from ocs_ci.ocs.node import get_nodes
+
+    if config.DEPLOYMENT.get("ocs_operator_nodes_to_taint", 0) > 0:
+        log.info("Verify worker nodes are tainted")
+        node_objs = get_nodes()
+        for node_obj in node_objs:
+            if node_obj.get("data").get("spec").get("taints")[0]["value"] != "true":
+                error_msg = f"The worker node {node_obj.name} is not tainted"
+                logging.error(error_msg)
+                raise ValueError(error_msg)
```

Comment:
What if `ocs_operator_nodes_to_taint != len(node_objs)` (assuming `get_nodes()` returns all worker nodes)?

Reply:
assuming get_nodes() returns all worker nodes --> correct
On ui_deployment, I dont test this scenario because I choose all worker nodes
![image](https://user-images.githubusercontent.com/61982127/150699318-504d1290-52d7-4e54-8318-5dea1f487b16.png)